frontend/src/concepts/pipelines/content/import/PipelineVersionImportModal.tsx

```diff
@@ -35,6 +37,8 @@ const PipelineVersionImportModal: React.FC<PipelineVersionImportModalProps> = ({
     setData,
     resetData,
   ] = usePipelineVersionImportModalData(existingPipeline);
+  const [{ items: versions }] = useAllVersionsByPipelineId(pipeline?.pipeline_id ?? '');
```

Comment:
same thing here, can we just add a filter for the name instead of fetching all versions